features/net/FEATURE_IPV4/lwip-interface/lwip-eth/arch/TARGET_STM/stm32xx_emac.c

```diff
@@ -448,3 +448,31 @@ void eth_arch_disable_interrupts(void)
 {
     NVIC_DisableIRQ(ETH_IRQn);
 }
+
+/** This returns a unique 6-byte MAC address, based on the device UID
+*  This function overrides hal/common/mbed_interface.c function
+*  @param mac A 6-byte array to write the MAC address
+*/
+void mbed_mac_address(char *mac) {
+    unsigned char ST_mac_addr[3] = {0x00, 0x80, 0xe1}; // default STMicro mac address
```

Comment:
yes, this is the official ST MAC address.
http://www.macmonster.co.uk/macoui=0080E1